devito/finite_differences/tools.py

```diff
@@ -86,8 +86,16 @@ def generate_fd_shortcuts(dims, so, to=0):
     from devito.finite_differences.derivative import Derivative
 
     def diff_f(expr, deriv_order, dims, fd_order, side=None, **kwargs):
-        return Derivative(expr, *as_tuple(dims), deriv_order=deriv_order,
-                          fd_order=fd_order, side=side, **kwargs)
+        # Spearate dimension to always have cross derivatives return nested
```

Comment:
@FabioLuporini this basically does `u.dxdy -> u.dx.dy`

Reply:
yeah, you could added it to the comment too `(e.g., u.dxdy -> u.dx.dy)`